### ISSUE TITLE
Changing [value] for multiple select only adds items to selection (does not remove any)

### ISSUE THREAD

**Issue by mactyr** [NejcZdovc/ng2-select2#141]
When I change the `[value]` of a select2 with the "multiple" attribute turned on, options in the new value that were not previously selected become selected, but all options that were previously selected also remain selected -- so my selection is the union of the old values and the new values, rather than just being the new values.

This also means that if I pass an empty array to `[value]`, nothing changes, whereas I would expect all options to be un-selected.

I believe the issue is in [setElementValue()](https://github.com/NejcZdovc/ng2-select2/blob/da679039bb3a048c3e1c499f48e805540520a4cd/lib/ng2-select2.component.ts#L166-L168), the innermost `if` needs an `else` to turn the selected property off when the option is not contained in the new value, like so:

```javascript
if (newValue.indexOf(option.value) > -1) {
  this.renderer.setElementProperty(option, 'selected', 'true');
} else {
  this.renderer.setElementProperty(option, 'selected', null);
}
```

**Comment by mactyr** [comment]
Or I suppose the whole block could just collapse to:

```javascript
this.renderer.setElementProperty(option, 'selected', newValue.indexOf(option.value) > -1 ? 'true' : null);
```

**Comment by fflores-origin** [comment]
yes actually adding that fix solves the problem, hope @NejcZdovc could update this plugin

**Comment by tyspice** [comment]
This issue seems to have been resolved in a more actively maintained fork: [ng-select2](https://www.npmjs.com/package/ng-select2)
